src/NServiceBus.Core/Reliability/Outbox/OutboxDeduplicationBehavior.cs

```diff
@@ -70,7 +70,7 @@ void DispatchOperationToTransport(IEnumerable<TransportOperation> operations, Co
                 var deliveryConstraints = deliveryConstraintsFactory.DeserializeConstraints(transportOperation.Options)
                     .ToList();
             
-                dispatchStrategy.Dispatch(dispatcher, message, routingStrategy, new AtLeastOnce(), deliveryConstraints, context);
+                dispatchStrategy.Dispatch(dispatcher, message, routingStrategy, new AtLeastOnce(), deliveryConstraints, context).GetAwaiter().GetResult();
```

Comment:
`DispatchOperationToTransport` operation to the web - sounds like we're doing heavy IO work here. Shouldn't method be async and call to strategy awaited?